src/index.js

```diff
@@ -1,10 +1,8 @@
-const fs = require('fs');
-
-const axe = fs.readFileSync('node_modules/axe-core/axe.min.js', 'utf8');
+import axe from 'axe-core';
 
 export const injectAxe = () => {
 	cy.window({ log: false }).then((window) => {
-		window.eval(axe);
+		window.eval(axe.source);
```

Comment:
Using `axe.source` will use the source from axe-core in this project's dependencies. Thus it'll be locked to axe-core ^4.0.2. 

To reproduce this issue, try installing this module with:
```
npm install --save-dev cypress axe-core@3.5.5
```

Once you run `cy.injectAxe() and cy.checkA11y()`, you can see the injected axe-core version is  4.0.2 (not 3.5.5).
Rather than using `axe.source`, I recommend using cy.readFile(), so the implementation will be:

```
export const injectAxe = () => {
	cy.readFile('node_modules/axe-core/axe.min.js').then((content) => {
		cy.window({ log: false }).then((window) => {
			window.eval(content);
		});
	})
};
```

I tested this implementation, and it successfully changes the axe-core version to 3.5.5 (for the scenario above)

Reply:
There's no bundled axe-core anymore in pull request, it's coming from your project's dependencies.

Reply:
Yes, I understand it's not bundled anymore. 
Perhaps, I encountered this issue because I tested this change with `npm link`.

Reply:
I've tried to copy the built file into my project's `node_modules`. Not sure it's much better than linking ;-)

Reply:
probably copy-pasting the built file is more suitable in this case. When I tested that way, this change works as expected 👍

Reply:
Cool, then I'm going to merge this and we could test the real thing and see what other issues we'll have ;-)